src/main/scala/com.snowplowanalytics.maxmind/iplookups/IpLookups.scala

```diff
@@ -114,29 +125,35 @@ class IpLookups(geoFile: Option[File] = None, ispFile: Option[File] = None, orgF
    * based on an IP address from one or
    * more MaxMind LookupServices
    *
-   * @param ip IP address
+   * @param ipS IP address
    * @return Tuple containing the results of the
    *         LookupServices   
    */
-  private def performLookupsWithoutLruCache(ip: String): IpLookupResult = {
+  private def performLookupsWithoutLruCache(ipS: String): IpLookupResult = {
```

Comment:
nit: not clear to me why `ip` morphed into `ipS`?

Reply:
Mostly to avoid confusion, the ip inside the function is an InetAddress while this one is a String

Reply:
gotcha